impl/maven-core/src/main/java/org/apache/maven/project/MavenProject.java

```diff
@@ -449,7 +417,7 @@ public Collection<SourceRoot> getSourceRoots() {
      * @since 4.0.0
      */
     public Stream<SourceRoot> getEnabledSourceRoots(ProjectScope scope, Language language) {
-        Stream<SourceRoot> s = sources.values().stream().filter(SourceRoot::enabled);
+        Stream<SourceRoot> s = sources.stream().filter(SourceRoot::enabled);
```

Comment:
Maybe s --> stream
When I see a single letter s variable, I think string, not stream.